package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "deckar01-task_list",
+  "version": "2.0.0-rc.1",
+  "description": "Markdown TaskList components",
+  "main": "app/assets/javascripts/task_list.coffee",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
Nope :). Good catch. I should probably allow starting the unit test runner from there.